src/Cooked/MockChain/BlockChain.hs

```diff
@@ -79,7 +80,7 @@ import Ledger.Index qualified as Ledger
 import Ledger.Slot qualified as Ledger
 import Ledger.Tx qualified as Ledger
 import Ledger.Tx.CardanoAPI qualified as Ledger
-import ListT
+import ListT hiding (null)
```

Comment:
Is it necessary? I saw no use of `null` in `Cooked.MockChain.BlockChain` on main nor in this PR.

If it were necessary, I think it would make it less confusing to provide an additive import list for definitions in `ListT` rather than hiding.

Reply:
It is not necessary indeed, good catch ! I must have used it at some point and then remove it and forget it in the imports. Thanks !